packages/scenes/src/components/VizPanel/VizPanel.tsx

```diff
@@ -266,7 +266,7 @@ export class VizPanel<TOptions = {}, TFieldConfig extends {} = {}> extends Scene
     //clear field config cache to update it later
     this._dataWithFieldConfig = undefined;
 
-    await this._loadPlugin(pluginId, newOptions ?? {}, newFieldConfig, true);
+    await this._loadPlugin(pluginId, newOptions ?? {}, newFieldConfig, refreshConfigs);
```

Comment:
don't like this function parameter
* Naming it confusing / in-accurate (only affects the isAfterPluginChange param)
* Don't think we need it. 

We could instead do 

```suggestion
    // If state.pluginId is already the correct plugin we don't treat this as plain user panel type change
    const isAfterPluginChange = this.state.pluginId !== pluginId; 
    await this._loadPlugin(pluginId, newOptions ?? {}, newFieldConfig, isAfterPluginChage);
```

Reply:
True, this is much better, also no need for a second PR in grafana for calling with that extra param 💯